package.json

```diff
@@ -31,16 +32,18 @@
     "zone.js": "0.8.26"
   },
   "devDependencies": {
-    "@angular/cli": "~6.0.7",
-    "@angular/compiler-cli": "^6.0.3",
     "@angular-devkit/build-angular": "~0.6.6",
     "@angular-devkit/build-ng-packagr": "~0.6.6",
+    "@angular/cli": "~6.0.7",
+    "@angular/compiler-cli": "^6.0.3",
     "@angular/language-service": "^6.0.3",
+    "@compodoc/compodoc": "^1.1.3",
     "@types/jasmine": "2.8.8",
     "@types/jasminewd2": "2.0.3",
     "@types/node": "8.9.5",
     "codecov": "3.0.2",
     "codelyzer": "4.2.1",
+    "event-stream": "^3.3.4",
```

Comment:
Is this required?

Reply:
Bizarrely, yes. I think compdoc upgraded just before added compodoc, and forgot a peer dependency, making the build fail. I'll retry with the latest version to see if it's fixed.